luigi2.h

```diff
@@ -3024,7 +3024,11 @@ int _UITableMessage(UIElement *element, UIMessage message, int di, void *dp) {
 	if (message == UI_MSG_PAINT) {
 		UIPainter *painter = (UIPainter *) dp;
 		UIRectangle bounds = element->bounds;
-		bounds.r -= UI_SIZE_SCROLL_BAR * element->window->scale;
+
+		if (table->vScroll->page < table->vScroll->maximum) {
```

Comment:
This could be simplified to be `bounds.r = table->vScroll->bounds.l;` I think?

Reply:
`UIScrollBar` has no member called bounds ?

Reply:
Sorry, `table->vScroll->e.bounds.l`

Reply:
yeah that won't work either, it breaks clicking and dragging the vertical scroll bars.